.github/workflows/ingest.yaml

```diff
@@ -28,12 +28,15 @@ jobs:
   ingest:
     permissions:
       id-token: write
+      contents: read
+      actions: write
```

Comment:
I don't think these permissions will flow into the reusable workflow as it sets (overwrites) its own permissions

https://github.com/nextstrain/.github/blob/38d02e1e39258a1609c6944bad9413af2462b33b/.github/workflows/pathogen-repo-build.yaml.in#L193-L194

These permissions are (I think) required for the GITHUB_TOKEN to be able to dispatch workflows

Reply:
Ah, the permissions/secrets/env are much simpler if we orchestrate the trigger completely within the GH Action workflow (e.g. [upload.yaml triggers a bunch of downstream workflows](https://github.com/nextstrain/seasonal-flu/blob/5a94edb61dfdb9249f61ff694781677853bdbe0a/.github/workflows/upload.yaml#L67-L94))